DSCResources/MSFT_SystemLocale/MSFT_SystemLocale.psm1

```diff
@@ -0,0 +1,148 @@
+#region LocalizedData
+$Culture = 'en-us'
+if (Test-Path -Path (Join-Path -Path $PSScriptRoot -ChildPath $PSUICulture))
+{
+    $Culture = $PSUICulture
+}
+Import-LocalizedData `
```

Comment:
Good catch!

As an alternative, we worked around the problem in xNetworking this way:
https://github.com/PowerShell/xNetworking/blob/dev/DSCResources/MSFT_xDnsClientGlobalSetting/MSFT_xDnsClientGlobalSetting.psm1#L1

This allowed us to keep the strings separate and still work on non en-us systems. Would this alternative solution work?